sycl/test-e2e/ESIMD/regression/dgetrf.cpp

```diff
@@ -9,7 +9,7 @@
 // REQUIRES: gpu
 // UNSUPPORTED: gpu-intel-gen9 && windows
 // UNSUPPORTED: cuda || hip
-// RUN: %{build} -I%S/.. -o %t.out
+// RUN: %{build} -I%S/.. -O1 -fno-inline-functions -o %t.out
```

Comment:
no-inline seems Ok for this test that gives timeout because of that.
But why -O1 is needed here?